packages/host/app/components/operator-mode/detail-panel.gts

```diff
@@ -88,89 +95,198 @@ export default class DetailPanel extends Component<Signature> {
+          <div class='details-panel'>
+            <header class='panel-header' aria-label='Details Panel Header'>
+              Details
+            </header>
+            <FileDefinitionContainer
               @fileURL={{@readyFile.url}}
-              @name={{@cardInstance.title}}
-              @fileExtension='.JSON'
+              @fileExtension={{this.fileExtension}}
               @infoText={{this.lastModified.value}}
               @actions={{array
                 (hash label='Delete' handler=@delete icon='icon-trash')
               }}
             />
-            <div>Adopts from</div>
-
-            <ClickableModuleDefinitionContainer
-              @fileURL={{this.cardType.type.module}}
-              @name={{this.cardType.type.displayName}}
-              @fileExtension={{this.cardType.type.moduleMeta.extension}}
-              @onSelectDefinition={{this.updateCodePath}}
-              @url={{this.cardType.type.module}}
-            />
           </div>
-        {{else if this.isModule}}
-          {{! Module case when visting, eg author.gts }}
-          <h3>In This File</h3>
-          {{#each @elements as |el|}}
-            <div
-              class='inheritance-chain {{if (this.isSelected el) "selected"}}'
-            >
-              <div>{{el.card.displayName}}</div>
-              <button {{on 'click' (fn @selectElement el)}}>Select</button>
-            </div>
-          {{/each}}
-          <h3>Inheritance Panel</h3>
-          <ModuleDefinitionContainer
-            @fileURL={{this.cardType.type.module}}
-            @name={{this.cardType.type.displayName}}
-            @fileExtension={{this.cardType.type.moduleMeta.extension}}
-            @infoText={{this.lastModified.value}}
-            @isActive={{true}}
-            @actions={{array
-              (hash label='Delete' handler=@delete icon='icon-trash')
-            }}
-          />
-          {{#if this.cardType.type.super}}
-            <div>Inherits from</div>
-            <ClickableModuleDefinitionContainer
-              @fileURL={{this.cardType.type.super.module}}
-              @name={{this.cardType.type.super.displayName}}
-              @fileExtension={{this.cardType.type.super.moduleMeta.extension}}
-              @onSelectDefinition={{this.updateCodePath}}
-              @url={{this.cardType.type.super.module}}
-            />
-          {{/if}}
-        {{else if this.isBinary}}
-          <FileDefinitionContainer
-            @fileURL={{@readyFile.url}}
-            @fileExtension={{this.fileExtension}}
-            @infoText={{this.lastModified.value}}
-            @actions={{array
-              (hash label='Delete' handler=@delete icon='icon-trash')
-            }}
-          />
         {{/if}}
       {{/if}}
     </div>
     <style>
-      .container {
+      .header {
+        --boxel-header-text-size: var(--boxel-font-size-sm);
+        --boxel-header-padding: var(--boxel-sp-xs);
+        --boxel-header-text-size: var(--boxel-font-size-sm);
```

Comment:
this line is a duplicate of line 237. On Zeplin it looks like the font-size should be smaller: `var(--boxel-font-size-xs)`